lib/services/git-service.ts

```diff
@@ -55,6 +55,33 @@ export class GitService implements IGitService {
             });
     }
 
+    clonePageTemplate(pageName: string, flavor: string, templatesDirectory: string): Promise<string> {
+        const command = "git";
+        const commandArguments: Array<any> = ["clone"];
+
+        return new Promise((resolve, reject) => {
+            util.getPageTemplatesBaseUrl(flavor)
```

Comment:
Does it makes sense to move this method from util in git service?

Reply:
And make it private?

Reply:
May be as a private method - yes, It does not trigger any git related actions though.